dockerfiles/mlrun-api/requirements.txt

```diff
@@ -16,6 +16,6 @@ memray~=1.12; sys_platform != 'win32'
 aiosmtplib~=3.0
 # until https://github.com/fastapi/fastapi/issues/10360 is solved
 pydantic>=1,<2
-# ensure kfp is installed
 mlrun-pipelines-kfp-v1-8~=0.4.3
-grpcio~=1.70.0
+grpcio~=1.59.0; python_version <= "3.9"
```

Comment:
I guess this is to ensure other component using mlrun-api requirements wont get wrong grpcio - *but* confusion it is because mlrun-api *is* and only should *be* running with python 3.11